docs/docs/rest-api/public/api/v2/types/queue.raml

```diff
@@ -0,0 +1,35 @@
+#%RAML 1.0 Library
+uses:
+  apps: app.raml
+  pods: pod.raml
+types:
+  QueueDelay:
+    properties:
+      timeLeftSeconds?:
+        type: number
+        format: int32
+      overdue?:
+        type: boolean
+  QueueObject:
+    properties:
```

Comment:
validation of the example failed when I added a discriminator. plus, the
concrete types are distinct enough: they each have a different set of
required fields

On Thu, Sep 1, 2016 at 5:08 PM, Jason Gilanfarr <notifications@github.com>
wrote:

> In docs/docs/rest-api/public/api/v2/types/queue.raml
> <https://github.com/mesosphere/marathon/pull/4286#discussion_r77254047>:
>
> > @@ -0,0 +1,35 @@
> > +#%RAML 1.0 Library
> > +uses:
> > +  apps: app.raml
> > +  pods: pod.raml
> > +types:
> > +  QueueDelay:
> > +    properties:
> > +      timeLeftSeconds?:
> > +        type: number
> > +        format: int32
> > +      overdue?:
> > +        type: boolean
> > +  QueueObject:
> > +    properties:
>
> need a discriminator and discriminator value if you're going to explicitly
> list the base type (not required).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4286/files/fc0c21d4e8b2115ebfed400084bb196299106ad0#r77254047>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLK7L0PvQBOSyocFl326Hm20ZpZy3ks5qlz67gaJpZM4JzC-I>
> .
>